### DISCUSSION TITLE
How to set up a mail client

### DISCUSSION THREAD

**Discussion by DeDelner** [docker-mailserver/docker-mailserver#3237] [Q&A]
Hi, 

I finally got my docker mailserver up and running. I already tested if the server can receive mails and yes, it does. Now I would like to set up a mail client like Outlook for example. But I couldn't get it to set up. I made sure every port is exposed, even disabled the firewall. However no luck. Am I missing something?

It seems like Outlook cannot establish a connection, since the logs don't say anything about an incoming connection. However, when I use a port checker and test it on 465 and 993, then at least I get an smtp error in the logs.

**Comment by jayknyn** [comment]
Hi, we're happy to help but you'll have to provide a lot more information such as:
- your docker-compose file for docker-mailserver
- output of error logs
- how are you terminating TLS? are you using a reverse-proxy such as Traefik or Caddy? If so, share those configs.
- Have you set up your DNS records?

You can test your mailserver's access using openssl from another computer:
`openssl s_client -crlf -connect mail.domain.tld:993`
Then at the blinking cursor:
`01 LOGIN <email address> <password>`
If that succeeds, then your mailserver should be fine. Type `02 LOGOUT` to close the openssl session.
